docs/source/models/risk_effects/hemoglobin/index.rst

```diff
@@ -177,57 +177,38 @@ Use the modeling strategy described below for the following maternal disorders s
-
-Relative risk PAF values can be accessed via shared functions with the following call:
-
-.. code:: python
-
-  # return age- and cause-specific relative risk estimates for the hemoglobin risk factor
-  get_draws(release_id=33, # GBD 2023 for topic-specific work
-              # TODO: confirm whether we should use release ID 16 or 33 here (both return data)
-              # Decided to use 33 for now to be compatible with ID in the PAF call below
-            source='rr',
-            gbd_id_type='rei_id',
-            gbd_id=376, # hemoglobin
-            sex_id=2, # female for maternal disorders causes
-            year_id=2022, # NOTE: this call returns only one value for year ID, which is 2022. You do not need to specify a year_id, but specifying any value besides 2022 will result in a failed call
-            location_id=1, # NOTE: you do not need to specify a location_id. Specifying any location ID will return results specific to location_id=1
-            ) 
-
-  # return age- and cause-specific PAF estimates for the hemoglobin risk factor
-  paf = get_draws(release_id=33, # GBD 2023 for topic-specific work
-              source='burdenator',
-              version_id=393, # obtained from: https://hub.ihme.washington.edu/spaces/GBDdirectory/pages/229280354/GBD+2023+PAF+Burdenator+SEV+Calculator+Tracking
-              gbd_id_type='rei_id',
-              gbd_id=376, # hemoglobin
-              sex_id=2, # female for maternal disorders causes
-              year_id=2023, # specify year_id=2023 to match 2023 exposure distribution
-              location_id=location_ids, # specify location_id to match exposure distribution for that location
-              metric_id=2, # metric=percent (returns PAF rather than attributable counts)
-              measure_id=1, # deaths (filter to single measure)
-                # TODO: figure out why PAFs are different for YLLs and YLDs depsite no difference in RRs between fatal and nonfatal outcomes
-                # decided to go with deaths for now because those values are less likely to be complicated by sequela-level detail
-              )
-  # filter down to relevant most detailed causes
-  paf = paf.loc[paf.cause_id.isin([367, # maternal hemorrhage
-                                  368, # maternal sepsis
-                                  369, # maternal hypertensive disorders
-                                  #567, # depressive disorders
-                                    # TODO: determine why there are no PAFs for depressive disorders
-                                  ])]
+  Calculate custom PAFs for maternal disorders and link to relevant files when ready.
+
+  Hemoglobin RRs have 250 draws and PAfs have 100 draws. To avoid blocking implementation, for now, let's use the first 100 draws for hemoglobin relative risks and calculate PAFs for draws 0-99 then copy them 5 times to get to a total of 500 draws like we have already done for hemoglobin exposure.
```

Comment:
```suggestion
  Hemoglobin RRs have 250 draws, while hemoglobin exposure has only 100 draws. Like we have already done for hemoglobin exposure, we'll use only the first 100 draws for hemoglobin relative risks and copy them 5 times to get a total of 500 draws.
```

Since GBD PAFs are no longer relevant